app/lib/notifier.rb

```diff
@@ -24,7 +24,9 @@ def guiders_to_notify
 
   def notify_customer
     if appointment_details_changed?
-      CustomerUpdateJob.perform_later(appointment, CustomerUpdateActivity::UPDATED_MESSAGE)
+      unless appointment.in_the_past?
```

Comment:
Given how this is used now it might be better expressed as `#future?` (or a name to that effect) and then you combine the two checks above like `if appointment_details_changed? && appointment.future?`